lib/app.js

```diff
@@ -534,6 +537,10 @@ module.exports = {
 
             machineThrottle = userThrottle(config, 'machines');
             machines.mount(server, machineThrottle);
+
+            if (config.experimental_cloudapi_changefeed === true) {
```

Comment:
Only real reason is that it's new. If we are Ok with it, we could just turn it on by default. No strong position about this.

Reply:
I'm starting to think this might be okay to enable by default - since the changefeed is only used (initialized) if one uses the '/changefeed' API path - so it's sort of isolated already (i.e. existing users/code should not be causing this to run).